### PR TITLE
Use plotly.js browser script vs. npm module

### PR DESCRIPTION
**Pull request by gnestor**
Closes https://github.com/jupyterlab/jupyter-renderers/issues/21

Resolves an issue with bundling plotly.js using webpack: https://github.com/plotly/plotly.js#building-plotlyjs-with-webpack

### CODE REVIEW

File: packages/plotly-extension/src/index.tsx [hunk 1]
```diff
@@ -9,10 +9,31 @@ import {
...
+
+  /**
+   * Load plotly.js browser script.
+   */
+  export
+  function load(): void {
+    loading = true;
+    const script = document.createElement('script');
+    script.type = 'text/javascript';
+    script.text = require('raw-loader!../vendor/plotly-1.28.3.min.js');
```
Comment (gnestor) [inline packages/plotly-extension/src/index.tsx]: @blink1073 Ideally, we could just set `script.src` to the relative path of the plotly.js file, but unfortunately that won't work once it's all bundled with webpack. Using raw-loader, we can get the content of the plotly.js file and set `script.text`, but this seems a little hacky... 🤔

Comment (blink1073) [reply packages/plotly-extension/src/index.tsx]: You don't even need to vendor the file if you use the `raw-loader`, you can use `raw-loader!plotly/dist/plotly.min.js`.  I think that this approach is fine.

Comment (gnestor) [reply packages/plotly-extension/src/index.tsx]: Do you mean _raw-load_ it from a CDN?

Comment (blink1073) [reply packages/plotly-extension/src/index.tsx]: No, as a third party import that webpack will resolve.

Comment (blink1073) [reply packages/plotly-extension/src/index.tsx]: But we don't load the script until it is actually needed.

Comment (gnestor) [reply packages/plotly-extension/src/index.tsx]: Can you show me what you mean?

Comment (blink1073) [reply packages/plotly-extension/src/index.tsx]: `script.text = require('!raw-loader!plotly/dist/plotly.min.js')`, where `plotly/dist/plotly.min.js` is the location of the file in `node_modules`.

Comment (gnestor) [reply packages/plotly-extension/src/index.tsx]: Oh ok, I like that better 👍

**Pull request merged**